Shared/Scripts/bxl.ps1

```diff
@@ -557,7 +557,7 @@ if (!$skipFilter){
 
     if ($Cache) {
         # Only build Cache code.
-        if ($LongRunningTest) {
+        if ($LongRunningTest -or $Vs) {

```

Comment:
I thought @SergeyTeplyakov fixed this a while ago

Reply:
Right now it doesn't include the LongRunningTest flag by default